Documentation/ApiOverview/LinkBrowser/Tutorials/_CustomLinkBrowser/_GitHubLinkHandlerFormatCurrentUrl.rst.txt

```diff
@@ -0,0 +1,15 @@
+.. Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets 
+.. Extracted from T3docs\Examples\LinkHandler\GitHubLinkHandler
+
+.. code-block:: php
+   :caption: Class T3docs\\Examples\\LinkHandler\\GitHubLinkHandler
+
+   class GitHubLinkHandler implements LinkHandlerInterface
+   {
+       public function formatCurrentUrl(): string
+       {
+           return 'https://gitub.com/' . $this->configuration['project'] . '/'
```

Comment:
```suggestion
           return 'https://github.com/' . $this->configuration['project'] . '/'
```

Reply:
We would need to change that in examples...